pkg/controllers/awsloadbalancercontroller/deployment_test.go

```diff
@@ -152,149 +153,6 @@ func TestDesiredArgs(t *testing.T) {
 	}
 }
 
-type testDeploymentBuilder struct {
```

Comment:
Be kind to your reviewer/s, don't move 150 lines of code to another place.  Now we have to review all the code you moved, to make sure it didn't change.

Reply:
Sorry about that, will try to minimize such moves.